topeft/modules/axes.py

```diff
@@ -60,4 +60,14 @@
         "variable": [0, 150, 250, 500],
         "label": r"Leading pt of pair from l+j collection (GeV) ",
     },
+    "ptz_wtau": {
+        "regular": (12, 0, 600),
+        "variable": [0, 150, 250, 500],
+        "label": r"pt of lepton hadronic tau pair (GeV) ",
+    },
+    "tau0pt": {
+        "regular": (12, 0, 600),
+        "variable": [0, 150, 250, 500],
+        "label": r"pt of leading hadronic tau (GeV) ",
+    },
```

Comment:
Should we consider adding the hadronic tau as the "l" in ptljmax? Maybe that's beyond the scope if this PR.

Reply:
@jlawrenc

Reply:
https://github.com/TopEFT/topeft/issues/454

Reply:
For the lj0pt, the tau is included when the tau_h_analysis flag is inititiated, but that is done in the analysis processor https://github.com/TopEFT/topeft/blob/ch_update_histeft/analysis/topeft_run2/analysis_processor.py#L851-L854

Reply:
Great! Sorry I missed that. What about ptZ? I see `ptz_wtau`, but does that get used anywhere currently?

Reply:
ptZ is untouched, so under the tau analysis, it specifically only uses e/mu leptons not taus. ptz_wtau is ptz with a tau lepton pair. Used as a kinematic fit varibale for some of the 2lss+1tau+onZ channels

Reply:
Ok, I see it now in the datacard post processing script.